task1.py

```diff
@@ -1,6 +1,55 @@
 """
 we have
-three classes, C=3
-four inputs D=4
+classes,        C=3
+inputs,         D=4
+training sets,  N=30
+test sets,      M=20
 
-"""
\ No newline at end of file
+"""
+
+import csv
+import matplotlib as plt
+import numpy as np
+
+#data importer
+def load_csv(filename):
+    #make data array
+    data = []
+    #open file
+    with open(filename, 'r') as file:
+        csv_reader = csv.reader(file)
+        #iterate through data and append
+        for row in csv_reader:
+            #add a weight at the end of each sample
+            data.append([float(val) for val in row]+ [1.0])
+    return data
```

Comment:
maybe this should be put in a separate file

Reply:
i dont think so